### PR TITLE
1.9.1 changelog

### PR DESCRIPTION
**Pull request by dpgraham**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,23 @@
+CHANGES IN VERSION 1.9.1 (from 1.9.0)
+===================================
+
+Appium 1.9.1 is a patch release
+
+#### iOS
+* Support Xcode 10 and iOS 12.0
+* Fix container type parsing
```
Comment (imurchie) [inline CHANGELOG.md]: The more important change is https://github.com/appium/appium-xcuitest-driver/commit/5e1ae26644dbb060c3b8594b2f4ac6bea03bc7fa

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,23 @@
...
+* Fix W3C format handling by receiveAsyncResponse command
+
+#### Android
+* Fix `getStrings` for no app
+* Return the current geolocation from `setGeoLocation`
+* Add /location to the list of NO_PROXY endpoints (UiAutomator2 only)
```
Comment (imurchie) [inline CHANGELOG.md]: Perhaps better expressed as fixing handling of getting geo location.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,23 @@
...
+* Add `mobile:mobileGetDeviceInfo` command (UiAutomator2 only)
+* Make 'by tag name' selector compatible with View Tag selector (Espresso only)
+* Convert mouse action events to touch actions (Espresso only)
+*
```
Comment (imurchie) [inline CHANGELOG.md]: Extra list element.

File: CHANGELOG.md [hunk 4]
```diff
@@ -5,17 +5,16 @@ Appium 1.9.1 is a patch release
 
 #### iOS
 * Support Xcode 10 and iOS 12.0
-* Fix container type parsing
+* Add a possibility to set container type for Simulator
```
Comment (mykola-mokhnach) [inline CHANGELOG.md]: .. while pushing and pulling files

File: CHANGELOG.md [hunk 5]
```diff
@@ -5,17 +5,16 @@ Appium 1.9.1 is a patch release
...
 * Fix W3C format handling by receiveAsyncResponse command
 
 #### Android
 * Fix `getStrings` for no app
 * Return the current geolocation from `setGeoLocation`
-* Add /location to the list of NO_PROXY endpoints (UiAutomator2 only)
+* Fix handling of getting geolocation endpoint (UiAutomator2 only)
```
Comment (mykola-mokhnach) [inline CHANGELOG.md]: this is not important and related to `Return the current geolocation from ` point

Comment (imurchie) [reply CHANGELOG.md]: Really? That commit (https://github.com/appium/appium-uiautomator2-driver/commit/c69adf7b6cceb591e181bdd4fa60abb2f2838c91) adds `GET` requests for `location` to the list. That seems different than setting the location.

**Comment by mykola-mokhnach** [comment]
@dpgraham https://github.com/appium/appium-base-driver/pull/259

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,3 +1,24 @@
+CHANGES IN VERSION 1.9.1 (from 1.9.0)
+===================================
+
+Appium 1.9.1 is a patch release
+
+#### General
+* Append the current session identifier to the protocol name prefix
```
Comment (mykola-mokhnach) [inline CHANGELOG.md]: this should also include multiprotocol handling fixes: https://github.com/appium/appium-base-driver/pull/258/files

Comment (mykola-mokhnach) [inline CHANGELOG.md]: ... name prefix in server logs

File: CHANGELOG.md (hunk 5 above)
Comment (mykola-mokhnach) [reply CHANGELOG.md]: Feel free to keep it if you like. For my taste it's a minor change

**Comment by dpgraham** [comment]
Changes pushed.

**Review by mykola-mokhnach — APPROVED** [review]

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,3 +1,25 @@
+CHANGES IN VERSION 1.9.1 (from 1.9.0)
+===================================
+
+Appium 1.9.1 is a patch release
```
Comment (KazuCocoa) [inline CHANGELOG.md]: If we stop supporting Xcode 8 in the next release, is it helpful to mention it?
=> https://github.com/appium/WebDriverAgent/pull/116

**Review by KazuCocoa — APPROVED** [review]

**Comment by mykola-mokhnach** [comment]
We could also mention the fixing of screenrecording endpoints:
https://github.com/appium/appium/issues/11279
https://github.com/appium/appium/issues/11276

**Comment by imurchie** [comment]
This release will have WinAppDriver 1.1 included. From Microsoft, the changes are:
```
New Features included in v1.1:
  1. Support for following Pointer Input methods:
      o Pointer Up
      o Pointer Down
      o Pointer Move
  2. Advanced Touch modifier attributes:
      o Pressure
      o Twist
      o Width
      o Height
  3. Advanced Pen modifier attributes:
      o Pressure
      o Twist
      o Tilt X & Y
      o Eraser
      o Barrel Button
  4. Interpolation support for Pen & Multi-Touch
```

Can you add these in a `Windows` section?

**Review by imurchie — APPROVED** [review]

**Pull request merged**